DB/Data/ChatbotContext.cs

```diff
@@ -11,7 +13,15 @@ public class ChatbotContext : DbContext
 
         protected override void OnConfiguring(DbContextOptionsBuilder builder)
         {
-            builder.UseInMemoryDatabase("chatDB"); 
+            if (Environment.GetEnvironmentVariable("DATABASE_URL") != null)
+            {
+                var m = Regex.Match(Environment.GetEnvironmentVariable("DATABASE_URL")!, @"postgres://(.*):(.*)@(.*):(.*)/(.*)");
```

Comment:
You can save the var of Environment.GetEnvironmentVariable("DATABASE_URL")
Then checking if its not null
This way, you dont need to to use the ! mark here